Week1/QA_PREP_EXERCISE.md

```diff
@@ -0,0 +1,17 @@
+As a preparation step for the upcoming Q&A, you will work through the following exercise. The expected result is a list of SQL `CREATE TABLE` statements that represent the tables you're being asked to design. We suggest you first think of the design of the tables (names, columns, types) and then proceed to write the SQL statements, that way you are thinking about the problem you're trying to solve (creating a database for a website), rather than the implementation first (creating a set of SQL statements).
+
+### Option 1
+
+Design the tables for a database that represents a website with food recipes. You don't need to write any specific queries at the moment, but the design should consider the need for some future queries.
+
+- Each recipe should have a name, a category, a list of ingredients, and a list of steps to follow to complete the recipe. Many recipes might share the same ingredients or the same list of steps (For example, cook pasta according to the package instructions).
+- The website will show a general page with all the recipes.
+- The website will also show recipes under a single category (Salads, Mexican, etc).
+
+### Option 2
+
+Design the tables for a database that represents a website for movies. You don't need to write any specific queries at the moment, but the design should consider the need for some future queries.
+
+- Each movie has a name, a category, a rating, a director, and a list of actors. An actor might be present in multiple movies and even be a director of others.
+- The website will show a general page with all the movies. The movies will be sorted by rating.
+- The website will show all movies under a single category (Comedy, drama, etc).
```

Comment:
I am not sure if we should introduce the word `website` in this exercise. We could just say that we would like to maintain a movies' database

Reply:
Sure, that's also fine!